exonum/src/events/tests.rs

```diff
@@ -292,9 +290,9 @@ fn test_network_reconnect() {
     t1.send_to(second, msg.clone());
     assert_eq!(t2.wait_for_message(), msg);
 
-    drop(t2);
-
+    t1.disconnect_with(second);
     assert_eq!(t1.wait_for_disconnect(), second);
+    drop(t2);
```

Comment:
Does it make sense to call `drop()` just before the variable goes out of scope? [The Rust docs](https://doc.rust-lang.org/std/mem/fn.drop.html) seem to tell that it is redundant in this case.

Reply:
Fxd.